src/main/java/pl/javastart/task/PalindromeService.java

```diff
@@ -4,6 +4,30 @@ public class PalindromeService {
 
     // nie zmieniaj sygnatury tej metody. Jest ona testowana w PalindromeServiceTest
     public boolean isPalindrome(int[] array) {
-        return false;
+
+        boolean isPal = true;
+
+        if (array == null) {
+            isPal = false;
+        } else if (array.length == 0) {
+            isPal = false;
+        } else {
+            int[] revArray = new int[array.length];
+            int j = array.length;
+
+            for (int k : array) {
+                revArray[j - 1] = k;
+                j = j - 1;
+            }
+
+            for (int i = 0; i < array.length; i++) {
+                if (array[i] != revArray[i]) {
+                    isPal = false;
+                    break;
+                }
+            }
```

Comment:
Takie rozwiązanie jest poprawne, ale nieoptymalne.
U siebie:
- tworzysz dodatkową tablicę
- iterujesz raz po całej tablicy, żeby skopiować elementy z pierwszej tablicy do drugiej
- iterujesz drugi raz po tablicy, żeby sprawdzić, czy tablice są sobie równe
Zamiast tego spróbuj zapisać 1 pętlę, w której będziesz porównywać kolejno 1 i ostatni element, 2 i przedostatni element itd. Wtedy da się to zrobić bez dodatkowej tablicy, dużo wydajniej